openmetadata-airflow-apis/openmetadata_managed_apis/api/routes/dag_status.py

```diff
@@ -0,0 +1,48 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Health endpoint. Globally accessible
```

Comment:
Every file has the same class description `Health endpoint. Globally accessible`

Reply:
looks like someone copy pasted this around 🙄